apps/www/_case-studies/epsilon3.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Epsilon3 digitize paper-based procedures in the space industry using telemetry data, simplifying testing and operations.
+description: Epsilon3 uses Supabase to help teams execute secure and reliable operations in an industry where project spend runs into the billions.
+author: rory_wilding
+author_title: Supabase
+author_url: https://github.com/kiwicopple
+author_image_url: https://avatars2.githubusercontent.com/u/10214025?s=400&u=c6775be2ae667e2acae3ccd347fed62bb3f5b3e7&v=4
+authorURL: https://github.com/kiwicopple
+logo: /images/case-studies/logos/epsilon3.png
+tags:
+  - supabase
+date: '28-10-2020'
+company_url: 'https://epsilon3.io'
+stats:
```

Comment:
Since we don't have any stat, should we remove this component from the old case studies?

Reply:
yea sure

Reply:
hidden for all for now